index.html

```diff
@@ -291,7 +291,7 @@ <h2>About me</h2>
       </div>
     </main>
 
-    <footer class="footer">
+    <section class="footer">
       <p>
```

Comment:
- [ ] A better semantic tag for this `section` here would be the `footer` tag. Do well to change that.